advanced/Scripts/webpage.sh

```diff
@@ -226,7 +226,8 @@ SetDNSServers() {
     IFS=',' read -r -a array <<< "${args[2]}"
     for index in "${!array[@]}"
     do
-        add_setting "PIHOLE_DNS_$((index+1))" "${array[index]}"
+        # Replace possible "\#" by "#". This fixes AdminLTE#1427
+        add_setting "PIHOLE_DNS_$((index+1))" "${array[index]//\\#/#}"
```

Comment:
Do you want to just pull out that particular pair or is it better to pull all potential escapes?

```suggestion
        add_setting "PIHOLE_DNS_$((index+1))" "${array[index]//\\}"
```

Reply:
From the web interface side, only `[ip]#[port]` is valid anyway, so it will only be a `#` that we need to unescape in this instance